src/mon/MonCommands.h

```diff
@@ -1190,6 +1190,24 @@ COMMAND("osd pool application get "
         "name=key,type=CephString,req=false",
         "get value of key <key> of application <app> on pool <poolname>",
         "osd", "r")
+COMMAND("osd pool stretch show "
+        "name=pool,type=CephPoolname",
+        "show all the stretch related information for the pool",
+        "osd", "r")
```

Comment:
So I currently can't think of other places where it would make sense to put the stretch info, I didn't put it in `ceph osd pool get` because we already have `ceph osd pool stretch show` which allows you to see all the information at once, unlike `ceph osd pool get` where you have to the run the commands multiple times to see all the values.